ingestion/src/metadata/ingestion/source/database/databricks/metadata.py

```diff
@@ -127,12 +127,14 @@ def get_columns(self, connection, table_name, schema=None, **kw):
             if db_name is not None:
                 rows = dict(
                     connection.execute(
-                        f"DESCRIBE {db_name}.{table_name} {col_name}"
+                        f"DESCRIBE {db_name}.{schema}.{table_name} {col_name}"
```

Comment:
@MilanBariya  can the `schema` be `None` in any case?

Reply:
@OnkarVO7 `schema` will not be `None`,  either it will come from config, or it will take from `database`.

Reply:
I would still add the handling for none just in case we have missed something

Reply:
Sure, let me handle that.

Reply:
nit - might be better to keep just `if db_name and schema` instead of passing the `schema is not None`. Here the DESCRIBE would fail for empty strings for schemas. Better to check for the false value of the whole string